:py:mod:`exceptions`
====================

.. py:module:: exceptions


Module Contents
---------------

.. py:exception:: CachePathNotSpecifiedError(message='Cache path is not specified in the JSON file!')


   Bases: :py:obj:`Exception`

   Exception raised when the cache path is not specified in the JSON file.


